schema/frogbot-schema.json

```diff
@@ -99,6 +99,11 @@
         "description": "Set to true to display all existing vulnerabilities, including the ones that were not added by the pull request.",
         "title": "Include All Vulnerabilities"
       },
+      "keepPreviousComments": {
+        "type": "boolean",
+        "description": "Keep old comments that were added by previous Frogbot scans on the pull request.",
+        "title": "Keep Previous Frogbot Comments"
+      },
```

Comment:
Let's uuse the variable name as in the workflows (JF_AVOID_PREVIOUS_PR_COMMENTS_DELETION --> AvoidPreviousPrCommentsDeletion).
Let's also use the same description.